.github/make_workflows.rb

```
@@ -0,0 +1,151 @@
+
+GCC_IMAGES = {
+  4 => "takei/libyu-dev:gcc4",
+  5 => "takei/libyu-dev:gcc5",
+  6 => "takei/libyu-dev:gcc6",
+}
+ALPINE_IMAGES = {
+  "gcc" => "takei/libyu-dev:alpine3-gcc",
+  "clang" => "takei/libyu-dev:alpine3-clang",
+}
+
+STANDARDS = [  nil, "c++11", "c++14", "c++17", "c++20", "c++23", ]
```

RuboCop (1.73)

[convention] Avoid comma after the last item of an array, unless items are split onto multiple lines.

(Style/TrailingCommaInArrayLiteral)